includes/options-page.php

```diff
@@ -5,8 +5,6 @@
  * @package Accessibility_Checker
  */
 
-use EDAC\Scans_Stats;
```

Comment:
Not needed. Also, I think calling this outside a class could cause global NS conflicts.